lib/type/integer.ex

```diff
@@ -7,3 +7,13 @@ end
 defmodule Antikythera.MilliSecondsSinceEpoch do
   use Croma.SubtypeOfInt, min: 0
 end
+
+defmodule Antikythera.MilliSecondsInGregorian do
+  @time_epoch_offset_milliseconds (:calendar.datetime_to_gregorian_seconds({{1970, 1, 1}, {0, 0, 0}}) * 1000)
+
+  def time_epoch_offset_milliseconds, do: @time_epoch_offset_milliseconds
```

Comment:
Even if there are no arguments, it is recommended to use parentheses.
```suggestion
  def time_epoch_offset_milliseconds(), do: @time_epoch_offset_milliseconds
```